src/Forms/Shared/Samples/Geometry/NearestVertex/readme.metadata.json

```diff
@@ -26,7 +26,7 @@
         "/net/latest/forms/sample-code/nearestvertex.htm"
     ],
     "relevant_apis": [
-        "GeometryEngine",
+        "Geometry",
```

Comment:
Metadata change should be reverted - sample is still using the `GeometryEngine` API.